File: pa3/DictionaryTest.java
class DictionaryTest {
public static void main(String[] args){
  Dictionary test = new Dictionary();


  test.insert("ab", "1");
  test.insert("v", "1");
  System.out.println(test.isEmpty());
  System.out.println(test.size());
  test.delete("ab");
  test.insert("p","3");
}
}

File: lab5/DictionaryTest.c
#include<stdio.h>
#include<stdlib.h>
#include<string.h>
#include"Dictionary.h"

#define MAX_LEN 180

int main(int argc, char* argv[]){
        Dictionary A = newDictionary();
        printf("%d",isEmpty(A));
        insert(A, "one","two");
        printf("%s",lookup(A, "one"));
        printDictionary(stdout, A);
        printf("%d",isEmpty(A));
        printf("%d",size(A));
        makeEmpty(A);
        printf("%d",size(A));

}
